test/matchUrl.test.js

```diff
@@ -1,4 +1,4 @@
-import matchUrl from '../js/matchUrl';
+const matchUrl = require('../js/matchUrl');
```

Comment:
for some reason the gulp plugin barfes if this uses  import... Imma remove gulp in a leter PR anyways 🙂